Dockerfile

```diff
@@ -21,18 +28,21 @@ RUN apk add --no-cache --update alpine-sdk \
     binutils \
     tar \
     protobuf-dev \
+    zip \
+&& curl -sfSLO ${PROTOC_URL} \
+&& unzip protoc-${PROTOC_VERSION}-linux-x86_64.zip -d /usr/local \
+&& rm /usr/local/bin/protoc /usr/local/readme.txt \
 && touch ~/.bashrc \
 && curl -sfSLO https://unofficial-builds.nodejs.org/download/release/${NODE_VERSION}/node-${NODE_VERSION}-linux-x64-musl.tar.xz \
 && tar -xf node-${NODE_VERSION}-linux-x64-musl.tar.xz -C /usr --strip 1 \
 && rm node-${NODE_VERSION}-linux-x64-musl.tar.xz \
 && curl -o- -L https://yarnpkg.com/install.sh | bash \
 && . ~/.bashrc \
-&&  cd /go/src/github.com/lightninglabs/lightning-terminal \
-&&  make install \
-&& go install -v -trimpath github.com/lightningnetwork/lnd/cmd/lncli \
-&& go install -v -trimpath github.com/lightninglabs/faraday/cmd/frcli \
-&& go install -v -trimpath github.com/lightninglabs/loop/cmd/loop \
-&& go install -v -trimpath github.com/lightninglabs/pool/cmd/pool
+&& git clone https://github.com/lightninglabs/lightning-terminal /go/src/github.com/lightninglabs/lightning-terminal \
+&& cd /go/src/github.com/lightninglabs/lightning-terminal \
+&& git checkout $checkout \
+&& make install \
+&& make go-install-cli \
```

Comment:
Remove the trailing `\` as it's causing the docker build to fail

Reply:
Argh, always these last minute changes that get me. Sorry, fixed.